web3-actions/tenderly.yaml

```diff
@@ -0,0 +1,37 @@
+account_id: ''
+actions:
+  moleculeprotocol/project:
+    runtime: v2
+    sources: actions
```

Comment:
not possible to just `.` and put all this in one folder? or pull up the package.json one level? Wrapping node projects like this is quite "dangerous" by the way (same with subgraphs) since npm walks up the directory hierarchy to resolve dependencies which might lead to unpredictable behaviour.